Gemfile

```diff
@@ -70,4 +71,9 @@ group :test do
   gem "capybara"
   gem "launchy"
   gem "simplecov"
+  gem "shoulda-matchers"
+  gem "orderly"
+  gem "faker"
+  gem "webmock"
+  gem "vcr"
```

Comment:
I am excited to get usage with the new gems we learned about `webmock` and `vcr`!!